Library/Source/Console.h

```diff
@@ -16,6 +16,10 @@ namespace babylon
         static void RegisterWarnOutput(MessageLogger output);
         static void RegisterErrorOutput(MessageLogger output);
 
+        static void Log(const char* message);
```

Comment:
Everything is static because the outputs in the console is static. When the runtime outputs something, it gets difficult to retrieve the corresponding console. Moreover, it makes sense for that to be related to the platform, not to the runtime.

Reply:
I'm not sure I agree that having the outputs should belong to the platform directly instead of the Runtime. As I'm currently thinking of the problem, messages only make sense when associated with the entity that sent them; where they go is less important than where they came from. Furthermore, a _lot_ of new risks get introduced when systems become static, including lingering state and cross-contamination in the case of multiple instances, all of which can be avoided by keeping things instance-specific. As I currently understand it, I really think the console (perhaps under a different name) should be an object owned by the Runtime which the Runtime's possessions can write to and the Runtime's owner can read from. The owning application might then choose to dump all the console outputs into the same platform-level output location; that seems to me like the overwhelmingly most likely use case. But I don't think that's the Runtime's decision to make; it should only have authority over and insight into its own messages.